CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt

```diff
@@ -0,0 +1,300 @@
+TM YB+1_S1 0 LB_RB+1_S1_BP1A 1580	CH0: W+1/RB1out/1 Backward 	CH1: W+1/RB2out/1 Backward 	CH2: W+1/RB1in/1 Backward 
```

Comment:
Looks like CondTools/RPC/data and CalibMuon/RPCCalibration/data has similar *.dat files (though different size)
```
-rw-r--r--. 1 22527 zh 1.2K Sep 23 19:39 ClSizeTot.dat
-rw-r--r--. 1 22527 zh 2.1M Sep 23 19:39 RPCDetId_Eff.dat
-rw-r--r--. 1 22527 zh 1.5M Sep 23 19:39 RPCDetId_Noise.dat
-rw-r--r--. 1 22527 zh  65K Sep 23 19:39 RPCTiming.dat
```
the CondTools/RPC/data/*.dat are not used any where (at least git grep does not show it). So should we drop these  CondTools/RPC/data/*.dat  files? If yes then I would say there is no need to have an extra data repo for  CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt  (which is 30K text file). 

@fthyssen by the way you should use full CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt  path in the CondTools/RPC/python/RPCTwinMuxLinkMapPopConAnalyzer_cfi.py file and then use FileInPath.